trl/trainer/ppo_trainer.py

```diff
@@ -906,6 +906,7 @@ def create_model_card(self, path: str, model_name: Optional[str] = "TRL Model")
             f.write(model_card_content)
 
     def _save_pretrained(self, save_directory: str) -> None:
+        self.accelerator.unwrap_model(self.model).save_pretrained(save_directory)
         self.model.save_pretrained(save_directory)
```

Comment:
We should remove the following line then, right?
```suggestion
```

Reply:
yes